packages/components/cascader-panel/src/index.vue

```diff
@@ -104,19 +106,32 @@ export default defineComponent({
       const { options } = props
       const cfg = config.value
 
-      manualChecked = false
-      store.value = new Store(options, cfg)
-      menus.value = [store.value.getNodes()]
-
-      if (cfg.lazy && isEmpty(props.options)) {
-        initialLoaded = false
-        lazyLoad(null, () => {
-          initialLoaded = true
+      const configTemp = Object.assign(config)
+      if (Object.keys(configTemp).includes('modelValue')) {
```

Comment:
This line always return false. Please check again.

Reply:
yes，This if is not necessary 。i removed it